contracts/tokenbridge/test/TestCustomTokenL1.sol

```diff
@@ -103,6 +131,63 @@ contract TestCustomTokenL1 is aeERC20, ICustomToken {
     }
 }
 
+contract TestOrbitCustomTokenL1 is TestCustomTokenL1 {
+    using SafeERC20 for IERC20;
+
+    constructor(address _gateway, address _router) TestCustomTokenL1(_gateway, _router) {}
+
+    function registerTokenOnL2(
+        address l2CustomTokenAddress,
+        uint256 maxSubmissionCostForCustomGateway,
+        uint256 maxSubmissionCostForRouter,
+        uint256 maxGasForCustomGateway,
+        uint256 maxGasForRouter,
+        uint256 gasPriceBid,
+        uint256 valueForGateway,
+        uint256 valueForRouter,
+        address creditBackAddress
+    ) public payable override {
```

Comment:
Could remove the payable if the compiler allows

Reply:
Compiler complains: `Error (6959): Overriding function changes state mutability from "payable" to "nonpayable".`